scripts/copyResearchDocs.ts

```diff
@@ -0,0 +1,46 @@
+import { execa } from 'execa';
+import { copyFile, cp, mkdir, rm, writeFile } from 'fs/promises';
+/**
+ * Copies the research documentation from TrackHAR. Since we do not translate the research docs (yet), we only copy them
+ * into the English section.
+ */
+
+(async () => {
+    // Get the current in-use version of TrackHAR
+    const trackharVersion = await execa('yarn', [
+        'list',
+        '--pattern',
+        'trackhar',
+        '--depth=0',
+        '--non-interactive',
+        '--no-progress',
+        '--json',
+    ]).then(({ stdout }) => JSON.parse(stdout).data.trees[0].name.split('@')[1] as string);
+
+    // eslint-disable-next-line no-console
```

Comment:
You can just disable the rule for the whole file: https://github.com/tweaselORG/appstraction/blob/b70a161dbce39737cbadd803314d7cc6702405d4/examples/ios-device.ts#L1